app/assets/scaffold/build/exclude_files.txt

```diff
@@ -30,7 +30,6 @@ codeception.yml
 deleted_files.txt
 index_dev.php
 modified_files.txt
-node_modules/*
```

Comment:
@escopecz I am not sure why do you want to include `node_modules`? since `npm-install` will install these files why do you want to have them included in the scaffold?

Reply:
The files must be present in the production package. We cannot ignore them. Or is this the wrong place?

Reply:
From my experience the node_modules/ is always created by npm install after checking out the repository or setting up the project. Its never added to the repo so it should be excluded everywhere (also I'm not sure what this scaffold really does).

Reply:
You are both right. Please review https://github.com/mautic/mautic/pull/12356